spaceship/lib/spaceship/connect_api/testflight/testflight.rb

```diff
@@ -191,11 +191,13 @@ def add_beta_groups_to_build(build_id: nil, beta_group_ids: [])
           test_flight_request_client.post("builds/#{build_id}/relationships/betaGroups", body)
         end
 
-        def create_beta_group(app_id: nil, group_name: nil, public_link_enabled: false, public_link_limit: 10_000, public_link_limit_enabled: false)
+        def create_beta_group(app_id: nil, group_name: nil, is_internal_group: false, public_link_enabled: false, public_link_limit: 10_000, public_link_limit_enabled: false)
           body = {
             data: {
               attributes: {
                 name: group_name,
+                isInternalGroup: is_internal_group,
+                hasAccessToAllBuilds: is_internal_group ? true : false,
```

Comment:
You could leave only `hasAccessToAllBuilds: is_internal_group,` and remove `? true : false,`

Reply:
Internal group can access all builds.

Reply:
I'm **super** torn on this `hasAccessToAllBuilds `... 

1. What this does it give the group access to builds that have already been released prior to the group being created
2. This is the default behavior of on the web

**HOWEVER**... `hasAccessToAllBuilds` is not a valid attribute in the official (App Store Connect API token) version of the API yet. This is only in the web version (Apple ID)... at least not in the docs or the Open API spec

**HOWEVER AGAIN**... this _does_ work with the API token even though its not in any of the docs forms. It appears to be a super secret hidden feature 😬 

I'm okay leaving this in but I'd like to leave a comment explain what this is and how it currently behaves...

```suggestion
                hasAccessToAllBuilds: is_internal_group ? true : false, # Undocumented of 2021-08-02 in ASC API docs and ASC Open API spec. This is the default behavior on App Store Connect and does work with both Apple ID and API Token
```